src/raftstore/store/fsm/batch.rs

```diff
@@ -322,6 +328,10 @@ impl<N: Fsm, C: Fsm, Handler: PollHandler<N, C>> Poller<N, C, Handler> {
                 }
             }
             self.handler.end(batch.normals_mut());
+            WORKER_POLL_HANDLE_BATCH_COUNTER
```

Comment:
Could you write `let v = XXX.with_label_values`?

Reply:
ok

Reply:
PTAL，thx